src/components/MenuBars/MenuBars.tsx

```diff
@@ -108,10 +109,11 @@ export const MenuBars = ({showPreviousColumn, showNextColumn, onPreviousColumn,
                 icon={RaiseHand}
                 onClick={toggleRaiseHand}
                 active={raisedHand}
+                hotkeyHint={TOGGLE_RAISED_HAND.toUpperCase()}
               />
             </li>
             <li>
-              <TooltipButton direction="right" label={t("MenuBars.settings")} onClick={showSettings} icon={SettingsIcon} />
+              <TooltipButton direction="right" label={t("MenuBars.settings")} onClick={showSettings} icon={SettingsIcon} hotkeyHint={SHOW_SETTINGS.toUpperCase()} />
```

Comment:
imo it looks really weird that the TooltipButtons now have two tooltips. Maybe we can add the hotkey hint right after the label (in a less visible font-color). What do you think?

Reply:
Good idea. This would also make some of the other requests obsolete.